server/api.js

```diff
@@ -13,8 +13,18 @@ router.get("/videos", async (_, res) => {
 	}
 });
 
-router.get("/health", async (_, res) => {
-	res.sendStatus(200);
+router.post("/videos", async (req, res) => {
+	if (!req.body.title) {
+		return res.status(422).json({ message: "Title field is required" });
+	}
+	if (!req.body.src) {
+		return res.status(422).json({ message: "src field is required" });
```

Comment:
It is good to see you using HTTP status codes other than 200 and thinking carefully about the right one to use. However, it's worth being aware that there are conventions - 400 bad request is usually  used for this situation. It makes it very clear that the problem is on the users side.

The message is a nice touch and would be much appreciated when debugging!